lab-katherine/README.md

```diff
@@ -0,0 +1,3 @@
+# a note app
+
+you can use it to take notes. press the add button.  ``` ¯\_(ツ)_/¯ ```
```

Comment:
you should try to get used to writing documentation, its likely you'll need to write accurate and clear docs of your code and apps in the future.